### ISSUE TITLE
How to compare time is the same day

### ISSUE THREAD

**Issue by ifangniu** [iamkun/dayjs#292]
like moment.js
`moment().isSame(date, "day")`

1. `dayjs().diff(date, 'day') === 0`
2. `dayjs().format('YYYYMMDD') === dayjs(date).format('YYYYMMDD')`

How to make an elegant comparison

dayjs().isSame(date, 'day')
It seems that second parameters are not supported?

**Comment by maeriens** [comment]
Not offering great help here - after digging  a bit in the code, as the docs point that `isSame()` can receive only one argument but is not really that descriptive; I found out that it checks the Unix Timestamp of the date compared by using  `.valueOf()`- so unless you are **that** precise with the comparison, it will always be *false*.

In terms of elegance, the first one looks nicer; but I am using the second one as I came across a small issue with a [false positive](https://runkit.com/embed/jvkclkmt3zgv) because it looks like `.diff(date, 'day')` checks that it's within the 24 hour range (not that I checked thoroughly, just hit that brickroad and swapped).

It would be nice to expand a bit the `isSame()` method to help in this cases.

**Comment by justin-schroeder** [comment]
Agreed, second argument should be supported to more closely conform to moment's api.

**Comment by probablyup** [comment]
>  I found out that it checks the Unix Timestamp of the dates

That's pretty different than what the documentation suggests:

> https://github.com/iamkun/dayjs/blob/master/docs/en/API-reference.md#is-same-issamecompared-dayjs

It seems that it should just be comparing the month day and year.
